tests/modules/cnvkit/batch/main.nf

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CNVKIT as CNVKIT_TUMORONLY } from '../../../../modules/cnvkit/batch/main.nf' addParams( options: [ 'args': '--reference reference.cnn --method wgs' ] )
```

Comment:
Since the process is in cnvkit/batch it should also be called this: `CNVKIT_BATCH` in the `main.nf` and imported here accordingly